<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpleDrawingWindow</class>
 <widget class="QWidget" name="simpleDrawingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	height:30px;
	background-color: rgb(42, 41, 160);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="horizontalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="modesBtn">
        <property name="text">
         <string>Modes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/modeBtnIcon.png</normaloff>../assets/modeBtnIcon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="horizontalFrame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	margin:0;
	border-radius:0px;
	background-color: rgb(51, 51, 51);
	width:100px;
	height:40px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="toolPane">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/penIcon.png</normaloff>../assets/penIcon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>T</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/textSelectionIcon.png</normaloff>../assets/textSelectionIcon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Upload</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/uploadBtnIcon.png</normaloff>../assets/uploadBtnIcon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../assets/exportBtnIcon.png</normaloff>../assets/exportBtnIcon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="verticalStackedWidget">
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="verticalStackedWidgetPage1" native="true">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
